includes/payments/functions.php

```diff
@@ -1225,6 +1224,34 @@ function edd_get_next_payment_number() {
 }
 
 /**
+ * Given a prefix, number and postfix, return the payment number,  incremented
+ *
+ * @since  2.4
+ * @param  string $prefix  The Payment Number Prefix
+ * @param  string $number  The formatted Current Number to increment
+ * @param  string $postfix The Payment Number Postfix
+ * @return string          The new formatted Payment number
+ */
+function edd_increment_payment_number( $prefix, $number, $postfix ) {
+
+	// Remove prefix and postfix
+	$number = str_replace( $prefix, '', $number );
```

Comment:
I don't think the prefix or postfix should be passed into this function.

Let's say a user decides to change their prefix or postfix after making some sales. The new prefix and postfix wouldn't be used after 2 more sales had been recored, since the first one would pull from the options table.

I vote we only store the number itself and then apply the prefix and post fix in `edd_get_next_payment_number()`.